client/odysseus/query/index.ts

```diff
@@ -35,3 +35,21 @@ export const useOddyseusSendMessage = (): UseMutationResult<
 };
 
 // TODO: We will add lately a clear chat to forget the session
+
+const odysseusGetChat = ( chat_id: string | null ) => {
```

Comment:
I'll suggest to change the name to something to indicate that it's a repeating call.

```suggestion
const pollOdysseusChatData = ( chat_id: string | null ) => {
```